SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
```

Comment:
These do not seem to be correct versions. We only have `0.x.y` so far.

Reply:
Right, this is simply a copy-paste of the boilerplate text. Will be changed according to the discussion in this thread.